org.envirocar.app/src/org/envirocar/app/views/carselection/CarSelectionAddCarFragment.java

```diff
@@ -95,6 +96,8 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle
             closeThisFragment();
         });
 
+        view.setOnClickListener(v -> hideKeyboard(view));
```

Comment:
It would be a great enhancement, if the keyboard would also close automatically, after the use has selected a value for hsn, tsn, manufacturer, model or construction year input fields :-)

Reply:
Yes. Thats a great idea. By closing the keyboard after seleting will resolve the issue of keyboard hiding the HSN/TSN feild to a certain extent. I will work on it :)